src/java/io/bazel/rulesscala/specs2/Specs2RunnerBuilder.scala

```diff
@@ -66,15 +87,15 @@ class Specs2ClassRunner(testClass: Class[_], testFilter: Pattern)
 
   private def specs2ExamplesMatching(testFilter: Pattern, junitDescription: Description): Option[String] =
     flattenDescription(junitDescription)
-      .filter(matching(testFilter))
```

Comment:
why the change from filter to find?

Reply:
Hmm, a leftover from trying different things. Will fix, thanks!

Reply:
Maybe we need another test?

Reply:
Perhaps. The test filter string doesn't currently support passing more than one test method, so currently there isn't a difference between `find` and `filter`. To be honest, I don't even know how to specify multiple test names in intellij...